spec/integration/task_cache_spec.rb

```diff
@@ -54,4 +54,14 @@
     run_cli(%w[task show] + flags)
     expect(original_mtime).not_to eq(File.mtime(project.task_cache_file))
   end
+
+  it 'clears existing cache with --clear-cache' do
+    run_cli(%w[module generate-types] + flags)
+    original_mtime = File.mtime(project.task_cache_file)
+    run_cli(%w[task show] + flags) #copied from above test to ensure 
+    #cache is behaving as expected, may not be necessary
+    expect(original_mtime).to eq(File.mtime(project.task_cache_file))
+    run_cli(%w[task show --clear-cache] + flags)
+    expect(original_mtime).not_to eq(File.mtime(project.task_cache_file))
```

Comment:
I tried something similar for line 178 in `plan_spec.rb` . It errored and said the file didn't exist. I'm not sure why? My best guess would be that `show task` puts something in the cache but `show plan` doesn't?

Reply:
Since we're just wanting to make sure the cache files are cleared with `--clear-cache`, it might be simpler to write a single unit test for the CLI that does the following:

1. Creates a temp project
2. Manually creates the cache files in the temp project
3. Runs a command with `--clear-cache` and expects `FileUtils` to receive `:rm` with each of the cache files _or_ checks that the cache files no longer exist